src/components/chats/Chat.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { Chat, ChatProps } from "components/chats/Chat"
+import { ChatMessage } from "components/chats/ChatMessage"
+import React, { useState } from "react"
+
+export default {
+  title:     "Chat",
```

Comment:
In storybook, let's organize the chat components into a folder; similar to how you did the accordion timeline component.

Reply:
Will do that.